qa-imovel-infobox/v2/qa-imovel-infobox.js

```diff
@@ -68,9 +68,14 @@ Polymer(
 		else{return "status_imovel " + status;}
 	},
 
-	statusImovelText: function(status) {
-		if(status === "edicaoSemUsuario" || status === "edicaoUsuarioNaoConfirmado"){return "edição"}
-		else{return status}
+	statusImovelText: function(status, suspensoAte) {
+		if (status === "edicaoSemUsuario" || status === "edicaoUsuarioNaoConfirmado") {
+			return "Edição";
```

Comment:
What about to keep this state capitalized, too?
```suggestion
			return "EDIÇÃO";
```

Reply:
It was uppercased before, but I've followed Invision and talked to Gioconda. The veredict: now this text is capitalized (only first character is uppercase).

Reply:
My bad!